muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/announcer/body/application/new-edit-announcement.tsx

```diff
@@ -260,37 +247,57 @@ class NewEditAnnouncement extends SessionStateComponent<NewEditAnnouncementProps
     nState[stateLocation] = newDate;
     (this.setState as any)(nState);
   }
+  
+  
   render(){
     let content = (closeDialog: ()=>any) => [
       //FOR DESIGN CHECK https://github.com/Hacker0x01/react-datepicker
       (<div className="container container--new-announcement-options" key="1">
-         <DatePicker selected={this.state.startDate} onChange={this.handleDateChange.bind(this, "startDate")}
-           locale={this.props.i18n.time.getLocale()}/>
-         <div className="text text--announcer-times-decor">-</div>                   
-         <DatePicker selected={this.state.endDate} onChange={this.handleDateChange.bind(this, "endDate")}
+        <div className="environment-dialog__form-element--wrapper">  
```

Comment:
Where is the BEM block element, it seems that there is only  BEM element "environment-dialog__form-element--wrapper" directly here. 

Also if we use modifiers we need to include the base element classname too.